Makefile

```diff
@@ -10,7 +10,7 @@ GOCACHE ?= $(shell echo ${PWD})/$(OUTPUT_DIR)/gocache
 # golang target architecture
 GOARCH ?= amd64
 # golang global flags
-GO_FLAGS ?= -v -mod=vendor
+GO_FLAGS ?= -v -mod=vendor -trimpath -ldflags=-s -ldflags=-w
```

Comment:
im not 100% sure, but seems `-ldflags=-s -ldflags=-w` != `-ldflags="-s -w"`. When using multiple `ldflags` it seems that:
- binary size reduction is not longer happening
- if the first iteration of the `ldflags` will have a wrong "flag" (_e.g `zz`_) , the build will not break, meaning that it seems the second `ldflags` definition takes precedence over the first one. 

@HeavyWombat can u check this pls?

Reply:
Oh, good catch. Let me check.